client/packages/common/src/ui/components/inputs/TextInput/numeric/PositiveNumberInput.tsx

```diff
@@ -28,7 +30,9 @@ export const PositiveNumberInput = React.forwardRef(
         InputProps={{
           sx: { ...sx, '& .MuiInput-input': { textAlign: 'right' } },
         }}
-        onChange={value => onChange(NumUtils.constrain(value, 1, max))}
+        onChange={value =>
+          onChange(NumUtils.constrain(value, Math.max(0, min), max))
```

Comment:
Can you revert this one too please - the usage of this input elsewhere expects a 1 minimum e.g. packsize which should never be zero. 

Perhaps add a parameter to set the minimum value?

Reply:
There is already a min parameter that defaults to 1 so it should behave like before (otherwise its a bug)